### PR TITLE
jg_it_2

### PR DESCRIPTION
**Pull request by elguapogordo**
all three percentage methods in iteration 2 have been fitted to stat_tracker.
i did not change when or where create_pct_data is called. that has been added to the stat_tracker methods.

### CODE REVIEW

File: lib/game_collection.rb [hunk 1]
```diff
@@ -40,7 +40,19 @@ def average_goals_per_game
   end
 
   def pct_of_total_games(outcome_type)
-    ((@pct_data[outcome_type] / @pct_data[:total_games].to_f) * 100).round(2)
+    (@pct_data[outcome_type] / @pct_data[:total_games].to_f).round(2)
```
Comment (elguapogordo) [inline lib/game_collection.rb:43]: *100 was not needed

File: lib/stat_tracker.rb [hunk 2]
```diff
@@ -144,4 +145,22 @@ def worst_offense
...
+    game_collection = GameCollection.new(@games_file)
+    game_collection.create_pct_data
+    game_collection.percentage_visitor_wins
+  end
+
+  def percentage_ties
+    game_collection = GameCollection.new(@games_file)
+    game_collection.create_pct_data
+    game_collection.percentage_ties
+  end
```
Comment (elguapogordo) [inline lib/stat_tracker.rb:164]: game_collection.create_pct_data should probably happen IN the file but we can save that for the spare refactoring time that probably won't exist. so it's not ideal but it IS functional, so i'm ok with it for now.

File: lib/game_collection.rb (hunk 1 above)
Comment (zachholcomb) [reply lib/game_collection.rb:43]: nice that should speed up the iteration

File: lib/stat_tracker.rb (hunk 2 above)
Comment (zachholcomb) [reply lib/stat_tracker.rb:164]: I totally agree, we will address that in a later refactor

**Pull request merged**